sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -764,3 +764,25 @@ def _append_original_pr_name_env(task_template):
     for step in task_template['taskSpec']['steps']:
         if step['name'] == 'main':
             _append_original_pr_name_env_to_step(step)
+
+
+def fix_big_data_passing_using_volume(workflow, pipeline_conf):
+    if workflow['spec'].get('workspaces'):
+        if pipeline_conf.data_passing_method._volume != None:
+            volume_dict = pipeline_conf.data_passing_method._volume.to_dict()
+            if volume_dict.get('persistent_volume_claim'):
+                for workspace in workflow['spec']['workspaces']:
+                    if workspace['name'] == workflow['metadata']['name']:
+                        workspace.pop('volumeClaimTemplate')
+                        temp_vc = {}
+                        for key, value in volume_dict.get('persistent_volume_claim').items():
```

Comment:
Inside of `if volume_dict.get('persistent_volume_claim')`, further `volume_dict.get('persistent_volume_claim')` instead of `volume_dict['persistent_volume_claim']` seems redundant. In fact, that call's result could be assigned to a variable and that variable could be checked for being `None` instead --- making it more clear that it will be used later after that initial check.

Reply:
done